src/collection-browser.ts

```diff
@@ -1165,8 +1185,18 @@ export class CollectionBrowser
    * This lets us keep track of queries so we don't persist data that's
    * no longer relevant.
    */
-  private get pageFetchQueryKey() {
-    return `${this.fullQuery}-${this.searchType}-${this.sortParam?.field}-${this.sortParam?.direction}`;
+  private get pageFetchQueryKey(): string {
+    const sortField = this.sortParam?.field ?? '';
+    const sortDirection = this.sortParam?.direction ?? '';
+    return `${this.fullQuery}-${this.searchType}-${sortField}-${sortDirection}`;
```

Comment:
Q: ok if returned string is `foo-bar--`?

Reply:
Ok and expected. Though I suppose an alternative would be to put a special `default` value in for the sort field/direction when they aren't defined.

Reply:
On second thought, it's better if these more explicitly convey something. So the no-sort case should be `foo-bar-none-none` now.